src/components/ui.js

```diff
@@ -550,27 +549,14 @@ AFRAME.registerComponent('ui', {
       material.needsUpdate = true;
     });
 
+    this.el.setAttribute('animation__showmessage', { dur: 500, property: 'components.ui.messagesMaterial.opacity', from: 0, to: 1, startEvents: 'showmessage' });
```

Comment:
this is brittle. I would add an opacity property to the UI component and the in the animation component do `property: ui.opacity`

Reply:
Similar to https://github.com/aframevr/a-painter/pull/262/files#diff-9557af62007f5be0a8d6c1f1e7bcef0e87749de6105efab34d294f360022ab7fR91